docs/ContainerRuntimeConfigDesign.md

```diff
@@ -71,6 +71,31 @@ spec:
    pidsLimit: 2048
 ```
 
+Make sure to add the label under `matchLabels` in the containerRuntimeConfig CR to the machineConfigPool config of the pools you
```

Comment:
"Make sure to add a `custom-crio` label under `matchLabels` in the ContainerRuntimeConfig CR and use that label in the MachineConfigPool config that you want the changes rolled out to."`

Reply:
Done! The label doesn't have to be `custom-crio`, but reworded it to something like what you said :)